torch/testing/_internal/common_methods_invocations.py

```diff
@@ -24227,8 +24227,7 @@ def skip(op_name, variant_name='', *, device_type=None, dtypes=None):
     return (op_name, variant_name, device_type, dtypes, False)
 
 
-def skipOps(test_case_name, base_test_name, to_skip):
-    all_opinfos = op_db
+def skipOps(test_case_name, base_test_name, to_skip, all_opinfos=op_db):
```

Comment:
The `all_opinfos` is a parameter now. Where does it serve?

Reply:
Let me roll back this change.